daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/scenario/TransactionProcessorTest.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package scenario
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class TransactionProcessorTest extends AnyWordSpec with Matchers {
```

Comment:
actually I have a better suggestion: just delete it completely (in a separate PR). If you look at `PrettyScenario`, we don’t actually display it and I don’t see why we would. So rather than doing work to test it, let’s delete it :)

Reply:
Follow on PR https://github.com/digital-asset/daml/pull/14263 will make these additional changes